src/tiled/mainwindow.ui

```diff
@@ -50,7 +50,7 @@
       <string>&amp;Recent Files</string>
      </property>
      <property name="icon">
-      <iconset resource="tiled.qrc">
+      <iconset resource="../../default_Imported_2fa4f2ef8a59764f_Desktop__69e7352954baa862/libtilededitor.877b2b3a/tilededitor.qrc">
```

Comment:
I didn't mean to change this...Qt Creator must have automatically done this when I opened mainwindow.ui ?

Reply:
Yeah, this is quite annoying. Qt Creator (or, the embedded Qt Designer) does not appear to handle nicely the fact that the resource files are generated by Qbs. I've just been excluding these changes from my commits...

Maybe I should work it around by removing resource file references from `.ui` files entirely and assigning resources, most commonly icons, from C++ code instead. Though that's going to be quite a few lines in case of the `MainWindow`. A patch for another time, in any case. :-)